code/controllers/subsystem/shuttles/supply.dm

```diff
@@ -216,6 +216,7 @@
 		/obj/item/device/radio/beacon,
 		/obj/machinery/the_singularitygen,
 		/obj/singularity,
```

Comment:
CentCom filed a complaint after some cargo techs created a singularity in the cargo bay and managed to get it into the shuttle and send it away.